network-api/networkapi/campaign/views.py

```diff
@@ -148,26 +148,12 @@ def signup_submission(request, signup):
     if cid is not None and cid != '':
         data['campaign_id'] = cid
 
-    # If we want to subscribe them through basket, do so, if not, subscribe them using SQS.
```

Comment:
Added in [6552](https://github.com/mozilla/foundation.mozilla.org/pull/6552/files#diff-5f0534998c1054cb2971d3f838cf3a37334e8d4c546b13088a734cee6213bde5R124) - no longer needed